lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -371,15 +389,41 @@ def resource_output_paths(resource_input_paths)
             end.uniq
           end
 
+          # Returns the framework output paths for the given input paths
+          #
+          # @param  [Hash<Array<Xcode::FrameworkPaths>>] framework_paths
+          #         The target's framework paths to map to output paths.
+          #
+          # @param  [Hash<Array<XCFramework>>] xcframeworks
+          #         The target's xcframeworks to map to output paths.
+          #
+          # @return [Array<String>] The embed frameworks script input paths
+          #
+          def embed_frameworks_input_paths(framework_paths, xcframeworks)
+            input_paths = []
+            framework_paths.each do |path|
```

Comment:
the nit of the nits I think you can do `input_paths = `framework_paths.each_with_object([]) { ... }`